cmd/placement/options/options.go

```diff
@@ -36,6 +37,14 @@ const (
 	defaultPlacementPort     = 50005
 	defaultReplicationFactor = 100
 	envMetadataEnabled       = "DAPR_PLACEMENT_METADATA_ENABLED"
+
+	envKeepAliveTime      = "DAPR_PLACEMENT_KEEPALIVE_TIME"
+	envKeepAliveTimeout   = "DAPR_PLACEMENT_KEEPALIVE_TIMEOUT"
+	envDisseminateTimeout = "DAPR_PLACEMENT_DISSEMINATE_TIMEOUT"
+
+	defaultKeepAliveTime      = 2 // in seconds
+	defaultKeepAliveTimeout   = 3 // in seconds
+	defaultDisseminateTimeout = 2 // in seconds
```

Comment:
Can we please make these Go duration strings to be consistent?

Reply:
I thought about it too, but defaulted to integers because values outside of the specified ranges (~1-10s) don't make sense and will be rejected, and also to follow what we already have in daprd (`app-health-probe-timeout`, `app-health-probe-interval`, `dapr-graceful-shutdown-seconds`).
I'm not opposed to changing it if you feel strongly about it, the benefit would be that people could use fractional values like 1.5 seconds.

Reply:
We can continue to have validation on ranges, regardless of the format.

As a project we are trying to converge on flags and options using unitised values- it's otherwise very confusing for consumers to attempt to guess or other wise have to think _more_ about the what unit the field is in. see https://github.com/dapr/dapr/commit/c920e26671cc75805fc7ca1b7bb50720538e4241 https://github.com/dapr/dapr/pull/7268/files

Reply:
Great, I'm on it!
I love that we're making a move to standardise this.